docs/src/pages/guides/cli-options.md

```diff
@@ -344,6 +355,14 @@ Like your app to be more chatty and honest revealing all its internal details? U
 monika --verbose
 ```
 
+## Verbose Cache
+
+Show (HTTP) cache hit / miss messages to log
+
+```bash
+monika --verbose-cache
```

Comment:
weird, i got no cache hit/miss log
<img width="734" alt="Screenshot 2024-06-11 at 13 39 03" src="https://github.com/hyperjumptech/monika/assets/5974862/d199cb25-ca33-4ec4-80d8-25401a9ec6b6">

<img width="1003" alt="Screenshot 2024-06-11 at 13 40 36" src="https://github.com/hyperjumptech/monika/assets/5974862/2110ce67-e4a1-41a6-8b2e-02880df692e5">

i use this `monika.yml`:
```
probes:
  - id: '1'
    name: GitHub
    description: Multiple
    requests:
      - url: https://github.com
        timeout: 7000
      - url: https://github.com/hyperjump
        timeout: 7000

db_limit:
  max_db_size: 1000000000
  deleted_data: 1
  cron_schedule: '*/1 * * * *'
```

Reply:
turns out it is a chaining request.

works on single request
![image](https://github.com/hyperjumptech/monika/assets/5974862/d2b45f31-9f00-41fb-bc40-22674e2011e3)


LGTM then...

Reply:
Added redundant documentation message for request chaining in `verbose-cache` for clearer explanation.